projects/storefrontlib/src/cms-components/configurator/commons/config-group-menu/config-group-menu.component.spec.ts

```diff
@@ -34,6 +34,10 @@ class MockRouter {
   public events = of('');
 }
 
+class MockConfiguratorGroupService {
+  public setCurrentGroup() {}
+}
+
```

Comment:
Why is a mock needed, the call is stubbed below, therefor no mock implementation is necessary.

Reply:
After a discussion with Christoph he told me that this is needed to reduce loading of store module. So this is fine